src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php

```diff
@@ -134,7 +134,9 @@ private function addTwigOptions(ArrayNodeDefinition $rootNode): void
                     ->example('Twig\Template')
                     ->cannotBeEmpty()
                 ->end()
-                ->scalarNode('cache')->defaultValue('%kernel.cache_dir%/twig')->end()
+                ->scalarNode('cache')
+                    ->setDeprecated('symfony/twig-bundle', '7.2', 'Setting the "%path%.%node%" configuration option is deprecated. It will be removed in version 8.0.')
```

Comment:
```suggestion
                    ->setDeprecated('symfony/twig-bundle', '7.3', 'Setting the "%path%.%node%" configuration option is deprecated. It will be removed in version 8.0.')
```